acct-group/thehive/thehive-0.ebuild

```diff
@@ -0,0 +1,9 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Group for TheHive"
+ACCT_GROUP_ID=-1
```

Comment:
can you check it out in your machine?
![Selection_155](https://user-images.githubusercontent.com/8144644/76012455-5d2fd300-5f27-11ea-9d95-860f4f96f642.jpg)